File: python/arreglos/SegundoEjercicio.py
import random

numeros = [0] * 10

# Llenar el arreglo
for i in range(10):
    numeros[i] = random.randint(1, 100)

# for clásico
print("Recorrido usando for clásico:")

for i in range(len(numeros)):
    print(numeros[i])

# for-each
print("\nRecorrido usando for-each:")

for numero in numeros:
    print(numero)

File: C#/arreglos/SegundoEjercicio.cs
using System;

class SegundoEjercicio
{
    static void Main()
    {
        int[] numeros = new int[10];
        Random r = new Random();
        for (byte i = 0; i < numeros.Length; i++)
        {
            numeros[i] = r.Next(1, 101);
        }
        Console.WriteLine("Recorrido for clasico");

        for (byte i = 0; i < numeros.Length; i++)
        {
            Console.WriteLine(numeros[i]);
        }
        Console.WriteLine("\nRecorrido con foreach:");
        foreach (int numero in numeros)
        {
            Console.WriteLine(numero);
        }
    }
}